rqes-data-classes/src/commonMain/kotlin/at/asitplus/rqes/enums/ConformanceLevelEnum.kt

```diff
@@ -0,0 +1,69 @@
+package at.asitplus.rqes.enums
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * CSC v2.0.0.2:
+ * The required signature conformance level
+ */
+@Suppress("unused")
+@Serializable
+enum class ConformanceLevelEnum {
```

Comment:
Why are all your enums called *Enum?

Reply:
Before moving them in their own folder it was used to keep order. Can remove if desired.

Reply:
d'accord